lms/djangoapps/onboarding/choices.py

```diff
@@ -0,0 +1,50 @@
+FUNCTIONS_LABELS = (
+    ('strategy_planning', 'Strategy and planning'),
+    ('leadership_governance', 'Leadership and governance'),
+    ('program_design', 'Program design and development'),
+    ('measurement_eval', 'Measurement evaluation and learning'),
+    ('stakeholder_engagement', 'External relations and partnerships'),
+    ('human_resource', 'Human resource management'),
+    ('financial_management', 'Financial management'),
+    ('fundraising', 'Fundraising and resource mobilization'),
+    ('marketing_communication', 'Marketing communications and PR'),
+    ('system_tools', 'Systems tools and processes'),
+)
+
+INTERESTS_LABELS = (
```

Comment:
Semantically these are not labels these are options or choices, any particular reason for calling them labels?